openapi/openapi/src/main/scala/endpoints4s/openapi/Urls.scala

```diff
@@ -83,6 +83,32 @@ trait Urls extends algebra.Urls {
   )
   type QueryStringParam[A] = DocumentedQueryStringParam[A]
 
+  //TODO: This is kinda ugly. But discriminated schemas make little sense on query strings?
+  //Stil maybe this should be done better, but I haven't yet fully understood the openapi schema to figure out how to deal with nesting here
+  private def unpackSchema[A](qsp:QueryStringParam[A]):List[Schema] = qsp.schema match {
+      case oneOf : Schema.OneOf => oneOf.alternatives match {
+        case enumerated: Schema.EnumeratedAlternatives => enumerated.alternatives 
+        case _ => List(qsp.schema)
+      }
+      case _ => List(qsp.schema)
+    }
+
+  def oneOfQueryStringParam[A,B](qspa: QueryStringParam[A], qspb: QueryStringParam[B]): QueryStringParam[Either[A,B]] = { 
+    DocumentedQueryStringParam[Either[A,B]](
+      schema = Schema.OneOf(
+        alternatives = Schema.EnumeratedAlternatives(
+          unpackSchema(qspa) ::: unpackSchema(qspb)
+        ),
+        description = None, //TODO: What to put here?
+        example = None, //TODO: What to put here?
+        title = None,//TODO: What to put here?
+      ),
+      isRequired = qspa.isRequired || qspb.isRequired, //TODO: This feels wrong?
```

Comment:
Here too, this is a signal that the signature of `oneOfQueryStringParam` is wrong, or that the type of `QueryStringParam` is wrong in the openapi interpreter (maybe it should be something else than the current `DocumentedQueryStringParam`).